code/add.py

```diff
@@ -37,7 +37,7 @@ def post_append_spend(message, bot):
     allocated_categories = helper.getCategoryBudget(chat_id)
     if selected_category not in allocated_categories.keys():
         helper.updateBudgetCategory(chat_id,selected_category)
-    helper.spend_categories.append(selected_category)
+    helper.spend_categories.insert(0,selected_category)
```

Comment:
any specific reason for this? seems to do the same thing?

Reply:
^minor

Reply:
Created category appears at the top in the next step. Easier for the user to see that their category has been added and can select it.